ts/Grid/Core/Credits.ts

```diff
@@ -200,7 +163,25 @@ class Credits {
  * */
 
 namespace Credits {
+    /**
+     * Extends the grid classes with credits.
+     *
+     * @param GridClass
+     * The class to extend.
+     *
+     */
+    export function compose(
+        GridClass: typeof Grid
+    ): void {
+        addEvent(GridClass, 'afterRenderViewport', initCredits);
+    }
 
+    /**
+     * Callback function called before table initialization.
+     */
+    function initCredits(this: Grid): void {
+        new Credits(this, Credits.defaultOptions);
```

Comment:
I would append it to the code structure though. The DOM element can be accessed using`document.getElementById` anyway. 

I mean:
```ts
this.credits = new Credits(...)
```

Reply:
In this case, we keep credits in the object. So users can check options in the Lite version. Stian wouldn't like to have it like this.